src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+$text-main-color: #060b35;
+$text-secondary-color: #616070;
+$color-white: #fff;
+$color-gray: #f3f3f3;
+$color-blue-accent: #00acdc;
+$smooth: 300ms;
+
+.product-card {
+  border: 1px solid $color-gray;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: $smooth;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: $smooth;
+  }
+
+  &__description:hover {
+    cursor:default;
+    transition: $smooth;
+    color: #34568b;
```

Comment:
The `&__description:hover` selector should be placed after the `&__description` block to ensure that the hover styles are applied correctly. CSS specificity rules dictate that the order of styles matters, and placing the hover state after the default state ensures it is applied.